<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UncertaintyToolBar</class>
 <widget class="QToolBar" name="UncertaintyToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>43</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>UncertaintyToolBar</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <action name="action_ToggleTitleBar">
   <property name="icon">
    <iconset resource="../../libs/guibase/guibase.qrc">
     <normaloff>:/images/titlebar-on.svg</normaloff>:/images/titlebar-off.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle Title Bar</string>
   </property>
  </action>
  <action name="action_ToggleSettings">
   <property name="icon">
    <iconset resource="../../libs/guibase/guibase.qrc">
     <normaloff>:/images/settings.svg</normaloff>:/images/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle Settings</string>
   </property>
  </action>
  <action name="action_CalculateNewSubEnsemble">
   <property name="icon">
    <iconset resource="../../libs/guibase/guibase.qrc">
     <normaloff>:/images/paste.svg</normaloff>:/images/paste.svg</iconset>
   </property>
   <property name="text">
    <string>New Ensemble from selected Members</string>
   </property>
  </action>
  <action name="action_WriteFullDataFile">
   <property name="icon">
    <iconset resource="../../libs/guibase/guibase.qrc">
     <normaloff>:/images/save-all.svg</normaloff>:/images/save-all.svg</iconset>
   </property>
   <property name="text">
    <string>New Ensemble from selected Members</string>
   </property>
  </action>
  <addaction name="action_ToggleTitleBar"/>
  <addaction name="action_ToggleSettings"/>
  <addaction name="action_CalculateNewSubEnsemble"/>
  <addaction name="action_WriteFullDataFile"/>
 </widget>
 <resources>
  <include location="../../libs/guibase/guibase.qrc"/>
 </resources>
 <connections/>
</ui>
